outbox/build.gradle

```diff
@@ -22,6 +22,7 @@ dockerCompose {
 }
 
 dependencies {
+    implementation project(':contract')
```

Comment:
I thought the outbox module should not have dependencies on other valtimo modules. Is this required?

Reply:
Yes, it is required for `val userId: String = SecurityUtils.getCurrentUserLogin() ?: "System",
    val roles: String = SecurityUtils.getCurrentUserRoles().joinToString(),`

Reply:
We want to remove this dependency. Instead we want a bean that has a copy of these methods. This bean can be overridden in implementations if teams want to use a different spring authentication. `UserProvider` This needs to be an open class and the bean registration needs to use ConditionalOnMissingBean